cmd/list/list.go

```diff
@@ -314,28 +314,40 @@ func (l *listTask) listWorker(wg *sync.WaitGroup, bkt *s3.Bucket, jobs <-chan *J
 		// track duration + inflight requests
 		metrics.inflight.Add(1)
```

Comment:
This line needs to be in the loop, or it will be decremented more times than incremented.